lib/streamlit/elements/plotly_chart.py

```diff
@@ -156,6 +168,7 @@ def marshall(
     figure_or_data: FigureOrData,
     use_container_width: bool,
     sharing: SharingMode,
+    theme: Union[None, str],
```

Comment:
Instead of `str` you could use `Literal["streamlit"]` here for better type checks and help